docs/proposals/workload.md

```diff
@@ -0,0 +1,81 @@
+# Workload schema
+
+Example :
+
+```json
+{
+    "kind" : "workload",
+    "type": "container",
+    "name": "test-workload",
+    "uri": "nginx",
+    "ressources": {
+        "cpu": 1.0,
+        "ram": 2048,
+        "disk": 2,
+    },
+    "ports": [
+        "8080:80"
+    ],
+    "environment": [
+        "NODE_ENV=production"
+    ]
+}
+```
+
+## type
+
+**type :** string
+
+Type of the workload :
+
+- `"container"`
+- `"vm"`
+- `"binary"`
+
+## name
+
+**type :** string
+
+Name of the workload, should be unique in the controller’s database.
+
+## uri
+
+**type :** string
+
+The name/uri of the container image , or an URI to the vm image or binary executable.
+
+## ressources
+
+**type :** object/struct
+
+Definition of the maximum (container, binary) or allocated (vm) ressources.
+
+### ressources.cpu
+
+**type :** float
+
+CPU power, in number of cores allocated as a floating number.
```

Comment:
```suggestion
**type :** int

CPU power, in number of cores allocated as a integer.
```

@Roxxas96 Are we okay, on that ? We set the number of core instead of the ratio (milliCPU) ?

Reply:
Is it a multiple of milliCPU or a multiple of CPU cores ?

Reply:
@NilsPonsard I think we all agreed on the `int` type. Every one put the cup resource in milliCPU (integer) rather than ratio (float)